samples/setEventBasedHold.js

```diff
@@ -43,7 +43,7 @@ function main(
 
   async function setEventBasedHold() {
     // Optional: set a generation-match precondition to avoid potential race
-    // conditions and data corruptions. The request to upload is aborted if the
+    // conditions and data corruptions. The request to set metadata is aborted if the
     // object's generation number does not match your precondition.
     const options = {
       ifGenerationMatch: generationMatchPrecondition,
```

Comment:
Shouldn't object metadata updates/patches be conditionally idempotent on the metageneration, not the generation, or am I missing something here/for node?

Reply:
You are right, let me clean this up and open a separate PR.